custom-connectors/Public360/Artifacts/apiDefinition.swagger.json

```diff
@@ -0,0 +1,281 @@
+    "contact": {
+      "name": "360 support",
+      "url": "https://tieto.service-now.com/"
+      }
+  },
+  "host": "masterkubepublicnorlrg.dev.360online.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Biz/v2/api/call/SI.Data.RPC/SI.Data.RPC/FileService/CreateFile": {
+      "post": {
+        "summary": "Create File",
+        "description": "Create File",
+        "operationId": "CreateFile",
+        "parameters": [
+          {
+            "name": "hosturl",
+            "in": "query",
+            "type": "string",
+            "required": true
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "parameter": {
+                  "type": "object",
+                  "properties": {
+                    "Title": {
+                      "type": "string",
+                      "description": "Title",
+                      "title": ""
+                    },
+                    "DocumentNumber": {
+                      "type": "string",
+                      "description": "Document Number",
+                      "title": "Document Number"
+                    },
+                    "DocumentRecno": {
+                      "type": "string",
+                      "description": "Document Recno",
+                      "title": "Document Recno"
+                    },
+                    "Format": {
+                      "type": "string",
+                      "description": "Format",
+                      "title": ""
+                    },
+                    "Base64Data": {
+                      "type": "string",
+                      "description": "Base64Data",
+                      "title": "Data"
+                    }
+                  },
+                  "description": "parameter",
+                  "required": [
+                    "DocumentRecno",
+                    "DocumentNumber",
+                    "Format",
+                    "Title"
+                  ]
+                }
+              },
+              "default": {
+                "parameter": {
+                  "Title": "Test Case wednesday",
+                  "DocumentNumber": "20/00269-2",
+                  "DocumentRecno": "200096",
+                  "Format": "txt",
+                  "Base64Data": "test"
+                }
+              },
+              "required": [
+                "parameter"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {

```

Comment:
@Pranav2889 Please replace the `default` responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.